### ISSUE TITLE
[HATS iOS] [Data] Catch Basin cleaning pending activities not being closed out automatically during "Inspection or Cleaning"

### ISSUE THREAD

**Issue by joeschmit** [WSDOT/HATS#123]
We are finding that we have several catch basins with multiple pending cleaning activities, and when one is "cleaned" the remaining are not being closed out.  Trett believes this was working in the past as stated.

**Comment by jhudsonWA** [comment]
It was. I'll look at why it's not happening. Is it not working on both Web and iPad?

**Comment by joeschmit** [comment]
Trett will have to confirm

**Comment by jhudsonWA** [comment]
I found the problem. The auto close of multiple cleaning records is broke on both the iPad and Web. It would of broke when we switched from WSDOTFeatureNumbers to AssetReferenceText nearly two years ago..

WHERE  e.WsdotFeatureNumber = (select WsdotFeatureNumber from event where eventid = @EventID)

I’ve fixed the issue server side, but it won’t fix previous ones unless I run some complicated queries.
